### PR TITLE
Trivy ignore AVD-AWS-0102 for subnet NACLs

### PR DESCRIPTION
**Pull request by richgreen-moj**
## A reference to the issue / Description of it

Daily run of the Static Code Analysis job is failing...
https://github.com/ministryofjustice/modernisation-platform/actions/runs/11340947819/job/31538282534#step:4:3094

## How does this PR fix the problem?

Added an ignore on a few resources in the `vpc-nacls` module for `avd-aws-0102` as the rules are open to all ports by design.

As suggested, I've also removed the tfsec ignores as that is no longer in use.

## How has this been tested?

Status checks are passing.

## Deployment Plan / Instructions

Will this deployment impact the platform and / or services on it?

{Please write here}

## Checklist (check `x` in `[ ]` of list items)

- [X] I have performed a self-review of my own code
- [x] All checks have passed
- [ ] I have made corresponding changes to the documentation
- [ ] Plan and discussed how it should be deployed to PROD (If needed)

## Additional comments (if any)

{Please write here}

### CODE REVIEW

**Review by dms1981 — CHANGES_REQUESTED** [review]
Can you try removing the `ignore` from `core-vpc/vpc.tf`, and moving the `ignore` so that it's before the resource declaration down in the module please?
https://aquasecurity.github.io/trivy/v0.41/docs/configuration/filtering/#by-inline-comments
Since we've migrated from tfsec we can probably remove those tfsec ignores too 😄

**Review by dms1981 — DISMISSED** [review]
LGMT - thanks!

File: terraform/modules/vpc-nacls/main.tf [hunk 1]
```diff
@@ -35,7 +35,7 @@ resource "aws_network_acl" "protected" {
   )
 }
 
-#tfsec:ignore:aws-vpc-no-excessive-port-access tfsec:ignore:aws-ec2-no-public-ingress-acl
+#trivy:ignore:AVD-AWS-0102
```
Comment (ewastempel) [inline terraform/modules/vpc-nacls/main.tf:38]: it would be useful if we added the information why the warning is being suppressed

Comment (richgreen-moj) [reply terraform/modules/vpc-nacls/main.tf]: Comment added to trivy ignores

**Review by ewastempel — APPROVED** [review]
LGTM, thanks for adding the comments

**Pull request merged**